obspy/core/tests/test_waveform_plugins.py

```diff
@@ -256,13 +265,17 @@ def test_read_thread_safe(self):
         be all the same.
         """
         data = np.arange(0, 500)
-        start = UTCDateTime(2009, 1, 13, 12, 1, 2, 999000)
         formats = _get_default_eps('obspy.plugin.waveform', 'writeFormat')
         for format in formats:
+            start = UTCDateTime(2009, 1, 13, 12, 1, 2, 999000) 
             # XXX: skip SEGY and SU formats for now as they need some special
-            # headers.
-            if format in ['SEGY', 'SU', 'SEG2']:
+            # headers. Also skip GCF as format does not permitt fractional
+            # start time for sampling rates < 250
+            if format in ['SEGY', 'SU', 'SEG2','GCF']:
                 continue
+            elif format in ['GCF']:
+               # XXX: GCF format does not support fractional for sampling rates <= 250 Hz
```

Comment:
Oh ok, weird. Maybe we should just set a sampling rate in the whole test case then, maybe 300 Hz if that works

Reply:
Sorry, 300 Hz is not supported by GCF format (You can find the supported sampling rates as the keys in dict `_SPS_MAP` in `core.py`, but, even so the start time is restricted to specific values (basically at `n/fractional_time_denominator` where `fractional_time_denominator` can be found as the values in  dict `_SPS_MAP` in `core.py`)

Reply:
Ah ok, fine then!